pulp_container/tests/functional/api/test_repositories_list.py

```diff
@@ -120,6 +128,7 @@ def test_listing_repositories(self):
         self.assertEqual(repositories.json(), {"repositories": repositories_names})
 
 
+@unittest.skipIf(TOKEN_AUTH_DISABLED, "Token authentication is not enabled")
```

Comment:
hmm..to summarize our logic.. So when we have token_auth enabled only public repos are visible to users with 0 perms or anon user. But when we have token_auth disabled we expose everything because there is no concept of public or private repo.